lib/controllers/cats.js

```diff
@@ -0,0 +1,45 @@
+const { Router } = require('express');
+
+const Cat = require('../models/Cat');
+
+module.exports = Router()
+  .get('/:id', async (req, res, next) => {
+    try {
+      const cat = await Cat.getById(req.params.id);
+      res.json(cat);
```

Comment:
Before you return the cat it would be a good idea to check whether cat contains any data (assuming your model returns null if the query didn't contain any data). If it doesn't, you could forward the request on to the next middleware using `next()`

```js
const cat = await Cat.getById(req.params.id);
if (!cat) next();
res.json(cat);